images/check-driver/Makefile

```diff
@@ -0,0 +1,15 @@
+IMAGE = falcosecurity/falco-driver-check
```

Comment:
I guess this is intended to be published to the docker.io registry, right?
If so:
- please, use the FQN
- we have to create the repo in dockerhub
- we have to handle the auth (AFAIK, pushing to dockerhub is not yet implemented in test-infra)

Reply:
Furthermore, there's no job configured to automatically build and publish the image when a code change gets merged. We should do that in a similar way we already do for prow images.

Reply:
You're right, and I might need some help for adding this with Prow. I fixed the FQDN as requested.